robot_ws/src/cloudwatch_robot/src/cloudwatch_robot/monitor_obstacle_distance.py

```diff
@@ -24,42 +24,44 @@
 from sensor_msgs.msg import LaserScan
 from ros_monitoring_msgs.msg import MetricList, MetricData, MetricDimension
 
-class MonitorNearestObstacle(Node):
+class MonitorObstacleDistance(Node):
```

Comment:
Not really important, but I think the original name aligned more closely with what the node is doing.

Reply:
Yea could change the file name instead. Just wanted it consistent with class and file name like the others.